pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -205,6 +205,24 @@ def test_dataframe_constructor_with_dtype():
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("frame", [True, False])
+def test_astype_dispatches(frame):
+    data = pd.Series(DecimalArray([decimal.Decimal(2)]), name='a')
```

Comment:
can you add a comment here why this test is added here (instead of as generic extension test)